readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -299,12 +300,17 @@ def test_response_building(self):
                 'version_slug': version.slug,
             },
         )
+
         build = resp.data
-        self.assertEqual(build['state'], 'cloning')
-        self.assertEqual(build['error'], '')
-        self.assertEqual(build['exit_code'], 0)
-        self.assertEqual(build['success'], True)
-        self.assertEqual(build['docs_url'], dashboard_url)
+        self.assertEqual(build["state"], "cloning")
+        self.assertEqual(build["error"], "")
+        self.assertEqual(build["exit_code"], 0)
+        self.assertEqual(build["success"], True)
+        self.assertTrue(build["docs_url"].endswith(dashboard_url))
+
+        # Validate received URL (fails with ValidationError if incorrect value)
+        url_validator = URLValidator(schemes=["https"])
+        url_validator(build["docs_url"])
```

Comment:
I don't think we are using this pattern in our tests. I'm not sure how this works but we are not `assert` anything here. I suggest to check for:

```
self.assertTrue(build["docs_url"].startswith("https://"))
```

in the same way you are doing in the previous line for `.endswith` and avoid introducing something new here.

Reply:
okay